"Cannot find module 'jquery.autocomplete'" with npm and browserify

Simply, in my Browserify initialization script I have the following
```
require('jquery.autocomplete');
```

But I get:
```
jean@Jean-Thinkpad:~/dev/mediabong/www_2$ npm remove jquery.autocomplete
removed 1 package in 4.047s
jean@Jean-Thinkpad:~/dev/mediabong/www_2$ npm i jquery.autocomplete
+ jquery.autocomplete@1.1.0
added 1 package in 4.228s
jean@Jean-Thinkpad:~/dev/mediabong/www_2$ browserify  -t [ browserify-css --minify=true --output public/style/node_style.css ] public/js/browserify_me_to_node_modules.js -o public/js/node_modules.js
Error: Cannot find module 'jquery.autocomplete' from '/home/jean/dev/mediabong/www_2/public/js'
    at /usr/lib/node_modules/browserify/node_modules/browser-resolve/node_modules/resolve/lib/async.js:46:17
    at process (/usr/lib/node_modules/browserify/node_modules/browser-resolve/node_modules/resolve/lib/async.js:173:43)
    at ondir (/usr/lib/node_modules/browserify/node_modules/browser-resolve/node_modules/resolve/lib/async.js:188:17)
    at load (/usr/lib/node_modules/browserify/node_modules/browser-resolve/node_modules/resolve/lib/async.js:69:43)
    at onex (/usr/lib/node_modules/browserify/node_modules/browser-resolve/node_modules/resolve/lib/async.js:92:31)
    at /usr/lib/node_modules/browserify/node_modules/browser-resolve/node_modules/resolve/lib/async.js:22:47
    at FSReqCallback.oncomplete (fs.js:158:21)
```

And jquery.autocomplete.js is in my node_modules directory as expected (everything works with a bunch of other modules).

I also tried with no success 
```
require('jquery-autocomplete');
require('jquery.autocomplete.js');
```

And finally had to do this as workaround:
```
require('../../node_modules/jquery.autocomplete/jquery.autocomplete.js');
```

I found no "main" entry in your package.json, like ```"main": "build/jquery.nouislider-range-input.js",```. Is it normal?

--- a/.travis.yml
+++ b/.travis.yml
@@ -3,4 +3,4 @@ sudo: false
 before_install:
   - npm install -g phantomjs
 node_js:
-  - "4.2"
+  - "8"

--- a/package.json
+++ b/package.json
@@ -11,6 +11,7 @@
     "webdriverjs-helper": "3.0.0"
   },
   "license": "MIT",
+  "main": "jquery.autocomplete.js",
   "scripts": {
     "test": "./node_modules/.bin/mocha -R spec -t 60000 ./test/test"
   },

